en/html/README.md

```diff
@@ -167,10 +167,10 @@ Make sure you're in the `djangogirls` directory and let's tell `git` to include
 
 {% filename %}command-line{% endfilename %}
 ```
-$ git add --all .
+$ git add .
 ```
 
-> __Note__ `--all` means that `git` will also recognize if you've deleted files (by default, it only recognizes new/modified files). Also remember (from chapter 3) that `.` means the current directory.
+> __Note__ By default, `git add` will recognize if you've deleted files (by default, it only recognizes new/modified files). Also remember (from chapter 3) that `.` means the current directory.
```

Comment:
Should be mentioned that `git add` only behaves this way when given a path argument? Or could the note be dropped completely?

Reply:
@das-g   I just went to https://tutorial.djangogirls.org/en/html/ and read it and it is not needed actually if `--all` option is not mentioned.

So I think it would not make any difference if it is there or not